### PR TITLE
Fixes #20454 - change page by typing page number

### PR DESCRIPTION
**Pull request by lizagilman**

### CODE REVIEW

File: webpack/assets/javascripts/foreman_tools.js [hunk 1]
```diff
@@ -97,6 +97,11 @@ export function updateTable(element) {
   if (perPage !== undefined) {
     uri.setSearch('per_page', perPage);
   }
+
+  pageNum = $('#pagination1-page').val();
```
Comment (ohadlevy) [inline webpack/assets/javascripts/foreman_tools.js]: i might be missing something, but where does the pagination1-page comes from? why is it called "1"?

**Comment by lizagilman** [comment]
updated

**Comment by lizagilman** [comment]
can anyone review? @tbrisker / @timogoebel  maybe ?

File: config/initializers/will_paginate.rb [hunk 2]
```diff
@@ -23,8 +23,8 @@ def forward
       end
 
       def pages
-        current_page_input = tag(:input, '', class: 'pagination-pf-page', type: 'text', value: current_page, id: 'pagination1-page')
-        current_page_label = tag(:label, _('Current Page'), class: 'sr-only', for: 'pagination1-page')
+        current_page_input = tag(:input, '', class: 'pagination-pf-page', type: 'text', value: current_page, id: 'cur-page-num')
```
Comment (tbrisker) [inline config/initializers/will_paginate.rb]: The id here is `cur-page-num` but below you look for `#cur_page_num` so it fails to find the right element. The tests also use `cur_page_num` which is why they pass.

**Review by tbrisker — CHANGES_REQUESTED** [review]
Just a small typo, see inline

**Comment by tbrisker** [comment]
Pushed the typo correction to your branch, will merge if all green.

**Comment by tbrisker** [comment]
Tests passed, merging. Thanks @lizagilman !

**Review by tbrisker — APPROVED** [review]

**Pull request merged**